connectors/hubspot/element-templates/hubspot-outbound-connector.json

```diff
@@ -27,7 +27,7 @@
     ]
   },
   "id": "io.camunda.connectors.HubSpot.v1",
-  "version": 1,
+  "version": 2,
```

Comment:
Can you please create a versioned subfolder and move the version: 1 template there, as we want to keep track of all versions in our repository.